### PR TITLE
fix default ip, add help for env setting

### PR DESCRIPTION
**Pull request by NeoZhangJianyu**

### CODE REVIEW

File: start_notebook.sh [hunk 1]
```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
-
+echo "Note: This script will startup pyspark with jupyter nootbook"
 # Check environment variables
 if [ -z "${BIGDL_HOME}" ]; then
     echo "Please set BIGDL_HOME environment variable"
+    echo "For example:"
```
Comment (wzhongyuan) [inline start_notebook.sh]: the above line is sufficient enough, every programmer should know what does set environment variable mean . i don't think we need to add extra info

File: start_notebook.sh [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+    echo "  export BIGDL_HOME=/path/to/python2.7/site-packages/bigdl/share"
     exit 1
 fi
 
 if [ -z "${SPARK_HOME}" ]; then
     echo "Please set SPARK_HOME environment variable"
+    echo "For example:"
```
Comment (wzhongyuan) [inline start_notebook.sh]: same as above

File: start_notebook.sh (hunk 1 above)
Comment (NeoZhangJianyu) [reply start_notebook.sh]: When user install bigdl from source, it's known where is the BIGDL_HOME.
But when install bigdl by PIP, it's hard to know by normal user. 
There is no document to tell user where is the BIGDL_HOME.
I have such problem and I find the answer after check.
Here, I want to give some clue to user for the case by PIP installation.

File: start_notebook.sh (hunk 2 above)
Comment (NeoZhangJianyu) [reply start_notebook.sh]: Same as above, for pyspark installed by PIP.

File: start_notebook.sh (hunk 1 above)
Comment (wzhongyuan) [reply start_notebook.sh]: it's irrelevant, and also it's not about setting the env,  it's about how to find you bigdl_home path.

Comment (NeoZhangJianyu) [reply start_notebook.sh]: Yes.
In the script help print, I want to guide user for the question: how to find you bigdl_home path.
I don't know where is good place to hint user.
So I put it in this script.

Do you think where is better?
I could update this info in right place.

Comment (wzhongyuan) [reply start_notebook.sh]: different installations have different results, also users may build it from scratch. I would suggest we don't add it, the promoted message intends to remind user to set variable, that's all

Comment (NeoZhangJianyu) [reply start_notebook.sh]: OK, I will remove them.

**Pull request merged**